---
float32 raw_error
float32 optimized_error
float32 raw_variance
float32 optimized_variance
float32 max_raw_error
float32 max_optimized_error
float32 raw_count
float32 optimized_count
